src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -6679,6 +6679,7 @@ public Tensor where(Tensor condition, Tensor y)
                     return ToMetadataString();
```

Comment:
Doc comment needs update. ` /// The style to use -- either 'metadata,' 'julia,' or 'numpy'`